pydis_site/templates/wiki/base.html

```diff
@@ -0,0 +1,49 @@
+{% extends "base/base.html" %}
+{% load sekizai_tags i18n wiki_tags static %}
+
+{% block title %}
+  Wiki | {% block wiki_pagetitle %}{% endblock %}{% block wiki_site_title %}{% endblock %}
+{% endblock %}
+
+{% block head %}
+  {{ block.super }}
+
+  <script src="{% static "wiki/js/jquery-3.3.1.min.js" %}" type="text/javascript"></script>
+  <script src="{% static "wiki/js/core.js" %}" type="text/javascript"></script>
+  <script src="{% static "js/wiki/simplemde.min.js" %}" type="text/javascript"></script>
+
+  <link rel="stylesheet" href="{% static "css/wiki/simplemde.min.css" %}">
+  <link rel="stylesheet" href="{% static "css/wiki/simplemde-fixes.css" %}">
+  <link rel="stylesheet" href="{% static "css/wiki/style.css" %}">
+{% endblock %}
+
+{% block content %}
+  {% block site_navbar %}
+    {% include "base/navbar.html" with active_item=True dropdowns=True icon_weight="far" use_logo=True %}
+  {% endblock %}
+
+  {% block wiki_navbar %}{% endblock %}
+
+  {% block wiki_breadcrumbs %}
+    {% include "wiki/includes/breadcrumbs.html" %}
+  {% endblock %}
+
+  {% block wiki_body %}
+    <section class="section">
+      <div class="container">
+        {% wiki_messages %}
+
+        <div class="content">
+          {% block wiki_contents %}{% endblock %}
+        </div>
+      </div>
+    </section>
+  {% endblock %}
+
+  {% block site_footer %}
+    {% include "base/footer.html" %}
+  {% endblock %}
+
+  <script src="{% static "js/wiki/dropdown.js" %}" type="text/javascript"></script>
```

Comment:
That's the first reason people do it - the other is so that scripts that don't need to be loaded for the page to function don't block the rendering of the page.

We can use a JS event, though. It just means the page will take longer to render.

Reply:
naw, fuck it, leave it as is.